src/main/java/seedu/duke/parsermanager/ParseDeleteClient.java

```diff
@@ -21,14 +22,18 @@ public class ParseDeleteClient extends Parser {
     private final String commandDescription;
     private final ClientList clientList;
 
+
+    private static final int CORRECT_FLAG_POSITION = 0;
```

Comment:
This is a good use of meaningful names to indicate the correct flag position. Good job :)